include/shisen_cpp/node/shisen_cpp_node.hpp

```diff
@@ -21,25 +21,29 @@
 #ifndef SHISEN_CPP__NODE__SHISEN_CPP_NODE_HPP_
 #define SHISEN_CPP__NODE__SHISEN_CPP_NODE_HPP_
 
+#include "../utility.hpp"
```

Comment:
Don't include like this. Use ```shisen_cpp``` lib which was added from our cmake.

```suggestion
#include "<shisen_cpp/{PATH_TO_LIB}>"
```

Reply:
In my opinion, it's okay to use `../utility.hpp` since the `utility.hpp` lies in the same project as this file.